src/Evolution/Systems/GrMhd/GhValenciaDivClean/Tags.hpp

```diff
@@ -70,8 +70,8 @@ using primitive_grmhd_reconstruction_tags =
 /// \brief Tags sent for spacetime evolution. All tags sent are the
 /// reconstruciton+spacetime tags
 using spacetime_reconstruction_tags =
-    tmpl::list<gr::Tags::SpacetimeMetric<3>, GeneralizedHarmonic::Tags::Phi<3>,
-               GeneralizedHarmonic::Tags::Pi<3>>;
+    tmpl::list<gr::Tags::SpacetimeMetric<3>, GeneralizedHarmonic::Tags::Pi<3>,
```

Comment:
why is the order switched here?

Reply:
This is now the correct order. I have (or will have, lost track) a PR that factors this order into a tmpl::list so different places don't need to by synced.

Reply:
Derp, that's this PR. The list with Pi first is the order in the GH system and should be the one used...

Reply:
Okay... I've refactored this changed into the first commit :D